### PR TITLE
Tomcat Agent Migration to Asgardio Repo

### PR DESCRIPTION
**Pull request by gayashanbc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -277,7 +277,7 @@ to match yours.
 
     <filter>
         <filter-name>SAML2SSOAgentFilter</filter-name>
-        <filter-class>org.wso2.carbon.identity.sso.tomcat.server.SAML2SSOAgentFilter</filter-class>
+        <filter-class>SAML2SSOAgentFilter</filter-class>
```
Comment (darshanasbg) [inline README.md]: Whats the best practice to refer filter class name? With or without FQN

File: pom.xml [hunk 2]
```diff
@@ -24,21 +24,21 @@
...
     </description>
-    <url>http://wso2.org</url>
+    <url>http://asgardio.io</url>
 
     <scm>
-        <url>https://github.com/wso2-extensions/identity-agent-sso.git</url>
-        <developerConnection>scm:git:https://github.com/wso2-extensions/identity-agent-sso.git</developerConnection>
-        <connection>scm:git:https://github.com/wso2-extensions/identity-agent-sso.git</connection>
+        <url>https://github.com/asgardio/asgardio-tomcat-saml-agent.git</url>
+        <developerConnection>scm:git:ttps://github.com/asgardio/asgardio-tomcat-saml-agent.git</developerConnection>
```
Comment (darshanasbg) [inline pom.xml]: Typo: scm:git:ttps -> scm:git:https

File: pom.xml [hunk 3]
```diff
@@ -347,8 +347,8 @@
     </properties>
 
     <modules>
-        <module>features/org.wso2.carbon.identity.sso.agent.server.feature</module>
-        <module>components/org.wso2.carbon.identity.sso.tomcat.server</module>
+        <module>features/o.asgardio.tomcat.saml.agent.server.feature</module>
```
Comment (darshanasbg) [inline pom.xml]: Typo: o.asgardio.tomcat.saml -> io.asgardio.tomcat.saml

**Review by darshanasbg — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (gayashanbc) [reply pom.xml]: Resolved in https://github.com/gayashanbc/identity-agent-sso/pull/1/commits/6f69998146be62873fc13ecd7564b53c08d0f76f

File: pom.xml (hunk 2 above)
Comment (gayashanbc) [reply pom.xml]: Resolved in https://github.com/gayashanbc/identity-agent-sso/pull/1/commits/6f69998146be62873fc13ecd7564b53c08d0f76f

File: README.md (hunk 1 above)
Comment (gayashanbc) [reply README.md]: Prepended FQN in https://github.com/gayashanbc/identity-agent-sso/pull/1/commits/3ee12f935a59ce79bdb211aebcaaf7213a262327

**Comment by gayashanbc** [comment]
Closing this PR as the review process completed

**Pull request closed without merging**